public/app/features/plugins/admin/hooks/usePluginDetailsTabs.tsx

```diff
@@ -34,6 +34,13 @@ export const usePluginDetailsTabs = (plugin?: CatalogPlugin, pageId?: PluginTabI
         url: `${pathname}?page=${PluginTabIds.VERSIONS}`,
         active: PluginTabIds.VERSIONS === currentPageId,
       });
+      navModelChildren.push({
```

Comment:
Should we check that the plugin has a changelog before pushing it into the nav model?